get_started.md

```diff
@@ -2,42 +2,59 @@
+    - John: an employee with access to all rooms
+    - Jane: a visitor with limited room access
+    - A stranger: they are allowed inside the HQ building!
+- Event schedules for people and rooms
 
 ## Overview of the Sandbox
 
 The Sandbox UI consists of the following elements.
 
-### Simulator window
+### User interface window
 
-TODO
+The right side of the screen shows the controls for the Access Control simulation. Each person in the **People** column has several icons: **Badge**, **Face**, **Parking**, and **WiFi**. These icons are all scanning options: for example, clicking on somebody's **Badge** icon simulates a "badge scan" into a building.
+
+Every person has a drop-down menu of the buildings/rooms they can scan into. Click on that box to choose a building/room, then click on one of the scan icons to choose a way to scan into that building/room. Every drop-down menu starts with only the **HQ Building** because you must scan into a building before you can scan into a room.
+
+You can also control time! Click the fast-forward icon by **Location at time** to set the time, which is important for visitors like Jane who cannot enter a room until a certain time.
 
 ### Ruleset, DDL, and Output tabs
 
 There are three tabs on the left. These are windows that show:
 
-- Ruleset: Displays the Declarative C++ code that handles TODO. You can modify the code in this tab and rebuild the sample to see how the changes affect the way that the sample runs. Rulesets encode the "business logic" that TODO
-- DDL: Displays the DDL code that describes the layout of the TODO, the state of the TODO, and events the factory produces.  In short, the DDL represents the data model for the app.
-- Output: Displays details when building/running your program.
+- **Ruleset:** Declarative C++ code that handles all the "business logic" of building and room access. You can edit the code in this tab and rebuild the sample to see how it affects the way that the sample runs.
+- **DDL:** A database schema describing the "data model" of everything in this Access Control scenario.
+- **Output:** Displays details while building/running your program.
 
 ## Build and run the sample
 
-For the best user experience, run the Sandbox in full screen mode set to Full HD or higher resolution.
+For the best user experience, keep the Sandbox's browser maximized to full-screen mode and use 1080p or higher resolution.
 
 The Sandbox contains a complete sample for you to examine and modify.
 
 To build the sample, select the **Build** button located above the simulator window. The state of the button changes depending on what point in the build/run cycle that you are in.
 
-While the sample builds, the **Output** window displays the progress of the build.
-
-**Note**: It can take several minutes to build the sample. 
+While the sample builds, the **Output** window displays the progress of the build. **It can take several minutes** to build the sample.
```

Comment:
While the sample builds, the **Output** window displays the progress of the build.

**Note**: It can take several minutes to build the sample.

Let's keep those two lines separate and we shouldn't bold statements like that. For emphasis set them off as a note.